docs/src/main/sphinx/release/release-423.md

```diff
@@ -0,0 +1,95 @@
+# Release 423 (2 Aug 2023)
+
+## General
+
+* Add support for renaming fields with a `RENAME COLUMN` statement. ({issue}`16757`)
+* Improve performance of `INSERT` and `CREATE TABLE AS ... SELECT` queries. ({issue}`18212`)
+* Reduce default values for the `task.partitioned-writer-count` and
+  `task.scale-writers.max-writer-count` configuration properties to reduce the
+  memory requirements of queries that write data. ({issue}`18488`)
+* Disallow coercing `timestamp` types to `varchar` for historical dates. ({issue}`18004`)
+* Fix query planning failure due to dynamic filters in
+  [fault tolerant execution mode](/admin/fault-tolerant-execution). ({issue}`18383`)
+* Fix URI extraction functions (`uri_extract_...`) ignoring input which are not
+  a valid URI. Previously, the functions returned ``NULL`` in such cases. ({issue}`18253`)
+* Fix failure to calculate query statistics when a query contains
+  `WHERE ... IN (NULL)`. ({issue}`18328`)
+
+## JDBC driver
+
+* Add support for
+  [constrained delegation](https://web.mit.edu/kerberos/krb5-latest/doc/appdev/gssapi.html#constrained-delegation-s4u)
+  with Kerberos. ({issue}`17853`)
+* Add support for taking action when there are connection parameter validation
```

Comment:
What is meant here with "taking action" ?

Reply:
@nineinchnick and @wendigo  .. can you clarify what this means?

Reply:
It just means that exceptions are now more informative and actionable. instead of "invalid argument" you will get exception explaining the problem i.e. "argument a cannot be used when argument c is set"

Reply:
It is user facing but not in a way thar is covered by docs

Reply:
Also note that the release notes suggested in the PR were different:
> * Connection parameter validation errors are now actionable ({issue}`12587`)

Reply:
I reworded now .. please have a look